### ISSUE TITLE
manual deployment doesn't seem to pick up env variables defined during the build

### ISSUE THREAD

**Issue by silky** [appveyor/ci#593]
if you pick a build to manually deploy, it doesn't seem to pick up the env variables that were defined as part of that build when you, say, set the 'tag name' that the release will use

can this be fixed? that way i can define the version during the build itself, and not have to automatically deploy.

**Comment by FeodorFitsner** [comment]
Correct, when the build is over its environment variables are gone. Storing build environment variables in artifacts sounds like a feature request.

**Comment by silky** [comment]
well, some environment variable remain; like `APPVEYOR_REPO_TAG_NAME` for example.

but what i observed that although this variable is somehow set, and you can even refer to it in the "tag name" in the manual configuration, it doesn't take on a value that might be *set* during the build (which is something i tried)

i can completely understand this; but the essential idea - setting the build number during the build and still allowing a *manual* deployment might be nice.

in any case, thanks for appveyor, it's great!
